docs/release/updating-to-osg-36.md

```diff
@@ -21,9 +21,38 @@ yum update
 
 AI (Carl): https://opensciencegrid.atlassian.net/browse/SOFTWARE-4473
 
-#### osg-configure ####
+#### OSG-Configure ####
+
+The OSG 3.6 release series contains OSG-Configure 4, a major version upgrade from the previously released versions in the OSG.
+See the [OSG-Configure 4.0.0 release notes](https://github.com/opensciencegrid/osg-configure/releases/tag/v4.0.0)
+for an overview of the changes.
+Several configuration modules and options were removed or deprecated and CE configuration has been simplified;
+the update from version 3 to version 4 will require some manual changes to your configuration.
+
+To update OSG-Configure, perform the following steps:
+
+1.  Update the osg-configure packages:
+
+        :::console
+        root@host# yum update 'osg-configure*'
+
+2.  Merge any `*.rpmnew` files in `/etc/osg/config.d/`
+
+3.  Uninstall `osg-configure-gip` and `osg-configure-misc` if they are installed:
+
+        :::console
+        root@host# yum erase osg-configure-gip osg-configure-misc
+
+4.  If `/etc/osg/config.d/30-gip.ini.rpmsave` exists, merge its contents into `31-cluster.ini`
+
+5.  If you are on a CE, edit the `Site Information` configuration section (in `40-siteinfo.ini`) and add:
```

Comment:
```suggestion
1. Edit the `Site Information` configuration section (in `40-siteinfo.ini`) and add:
```

`osg-configure` in 3.6 is just a CE package, at least for now. We should also tell users to only add this if they don't have it already set.